README.md

```diff
@@ -1 +1,60 @@
 # st21-chestX
+<b>ChestX</b> is a <b>TechLabs Berlin</b> project that aims to detect COVID-19 infection by processing chest X-ray images. To support medical decission making, the ChestX team wants to applying AI technics for object detection & classification, as well as data science methodes. Metadata provieded alongside the X-ray images can be analysed to providing doctors with more meaningful diagnostic assistance.
+
+## Data set
+The COVID-19 image data collection is a public open dataset of chest X-ray and CT images of patients which are positive or suspected of COVID-19 or other viral and bacterial pneumonias. The project was started and is maintained by Joseph Paul Cohen, Paul Morrison, and Lan Dao from the University of Montreal.
+Please, watch the following [video](https://www.youtube.com/watch?v=ineWmqfelEQ) for a quick project introduction, or check out there [paper](https://arxiv.org/abs/2003.11597) for more details about the [data set](https://github.com/ieee8023/covid-chestxray-dataset).
+
+This COVID-19 data set is linked as submodule to this git project. <br>Run the following code to download the submodule:
+```git submodule update --init --recursive```
+<br>To update the submodul run:
+```git submodule update --remote --merge```
+
+## Related research & solutions 
+Referencingthe COVID-19 image data collection reposetory, some ML models were developed alongside the data collection process and constitute the main purpose of this research project. 
+
```

Comment:
Referencing the COVID-19 image data collection repository, some ML models were developed alongside the data collection process and constitute the main purpose of this research project.

Reply:
corrected the typos